### PR TITLE
Fixed some typos in ractor.md

### PR DESCRIPTION
**Pull request by ashelangovan**
While going through the doc, found some minor grammatical errors and fixed them.

### CODE REVIEW

File: doc/ractor.md [hunk 1]
```diff
@@ -19,9 +19,9 @@ You can make multiple Ractors and they run in parallel.
 
 Ractors don't share everything, unlike threads.
 
-* Most of objects are *Unshareable objects*, so you don't need to care about thread-safety problem which is caused by sharing.
+* Most of the objects are *Unshareable objects*, so you don't need to care about thread-safety problem which is caused by sharing.
```
Comment (marcandre) [inline doc/ractor.md:22]: I'd write "Most objects"...

**Comment by marcandre** [comment]
Thanks for the PR. I'd merge it, but CI seems down and I can't. I suggest doing the one small change I noted and force pushing, maybe that will restart the CI

**Comment by ashelangovan** [comment]
@marcandre Made the required changes but still, CI seems down.

**Comment by marcandre** [comment]
CI is back :-) Merged, thanks!

**Review by phykos — APPROVED** [review]

**Pull request merged**